devel/legacy-support/Portfile

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        cjones051073 macports-legacy-support 9950debbadb2074c7eee25d62ece352e1baed32c
+name                legacy-support
+version             20181027
+revision            0
+categories          devel
+platforms           darwin
+maintainers         {jonesc @cjones051073} {kencu @kencu} openmaintainer
+license             GPL-2
+
+description         Installs wrapper headers to add missing functionality from legacy OSX versions.
```

Comment:
Does this provide a different set of functions on each OS version? It would be best not to shadow anything that the OS already provides.

Reply:
As @kencu says, Both the wrapped headers and the library files providing the implementations are protected such that they never implement anything the os provides.